frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/pipeline-as-code.ts

```diff
@@ -13,19 +13,37 @@ import {
 } from '@console/dev-console/integration-tests/support/pages';
 import { pipelineActions, pipelineTabs, repositoryDetailsTabs } from '../../constants';
 import { createRepositoryPO } from '../../page-objects';
-import { pipelinesPage, repositoryDetailsPage, repositoriesPage } from '../../pages';
+import { repositoryDetailsPO } from '../../page-objects/pipelines-po';
+import {
+  pipelinesPage,
+  repositoryDetailsPage,
+  repositoriesPage,
+  pipelineRunDetailsPage,
+} from '../../pages';
 import { actionsDropdownMenu, tableFunctions } from '../../pages/functions/common';
 
 Given('user has installed pipelines as code', () => {
   // Steps to install pipeline as code feature : https://gist.github.com/chmouel/272df2cfbeef5606ca36d29a9a2d2f96
-  cy.exec(
-    `oc apply -f https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/release-0.1/release-0.1.yaml`,
-    {
-      failOnNonZeroExit: false,
-    },
-  );
-  cy.exec(`oc expose service el-pipelines-as-code-interceptor -n pipelines-as-code`, {
-    failOnNonZeroExit: false,
+
+  cy.get('body').then(($body) => {
+    if ($body.find('[data-test-id="horizontal-link-Repositories"]').length > 0) {
```

Comment:
there is a [page object](https://github.com/openshift/console/blob/468f66b39f70f89c143c43c0e2267c1e5076bf87/frontend/packages/pipelines-plugin/integration-tests/support/page-objects/pipelines-po.ts#L233) for this selector, we should use it instead